### PR TITLE
blog post: Scala 3 roadmap for 2024

### PR DESCRIPTION
**Pull request by SethTisue**
pretty much just links to https://virtuslab.com/blog/technology/scala-3-roadmap-for-2024/ — very little fresh wording

### CODE REVIEW

**Comment by SethTisue** [comment]
Nothing controversial here I don't think, so although I requested a couple of reviewers, I'll merge it today regardless if I don't hear anything back.

The Virtus blog post was already tweeted from https://twitter.com/scala_lang/status/1754902373539914047

**Review by Gedochao — APPROVED** [review]
LGTM

File: blog/_posts/2024-03-15-scala-3-roadmap-2024.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+We call your attention to the following blog post from VirtusLab:
+
+* https://virtuslab.com/blog/technology/scala-3-roadmap-for-2024/
+
+It lays out the Scala 3 team's plans for Scala 3 this year.
+
+Topics covered include the timing, contents, and compatibility stories of future Scala 3 versions. There's also specifics about plans to make improvements in build times, IDE support, and elsewhere.
+
+**Reminder** The Scala Center also regular publishes roadmaps.
```
Comment (sjrd) [inline blog/_posts/2024-03-15-scala-3-roadmap-2024.md:17]:
```suggestion
**Reminder** The Scala Center also regularly publishes roadmaps.
```

**Review by sjrd — APPROVED** [review]

**Comment by SethTisue** [comment]
fixup 6df641b1a7e3f8bd97349cac69e88f8400c682f8 
live at https://www.scala-lang.org/blog/2024/03/15/scala-3-roadmap-2024.html

**Pull request merged**